src/App.tsx

```diff
@@ -1,18 +1,31 @@
-import { PeoplePage } from './components/PeoplePage';
-import { Navbar } from './components/Navbar';
+import {
+  Routes,
+  Route,
+  Navigate,
+} from 'react-router-dom';
+import { HomePage } from './pages/HomePage';
+import { NotFoundPage } from './pages/NotFoundPage';
+
+import { PeoplePage } from './pages/PeoplePage';
+import { NavBar } from './components/Navbar';
 
 import './App.scss';
 
 export const App = () => {
   return (
     <div data-cy="app">
-      <Navbar />
+      <NavBar />
 
       <div className="section">
         <div className="container">
-          <h1 className="title">Home Page</h1>
-          <h1 className="title">Page not found</h1>
-          <PeoplePage />
+          <Routes>
+            <Route path="/" element={<HomePage />} />
+            <Route path="/home" element={<Navigate to="/" replace />} />
+            <Route path="/people">
+              <Route path=":peopleId?" element={<PeoplePage />} />
+            </Route>
+            <Route path="*" element={<NotFoundPage />} />
+          </Routes>
```

Comment:
It's better to use `Outlet` component and move your routes to a separate file https://reactrouter.com/en/main/components/outlet